packages/web/src/dev/Engage.tsx

```diff
@@ -206,6 +108,136 @@ const Aside = React.memo(function _Aside({
   )
 })
 
+export function Contribute() {
+  const { t } = useTranslation(NameSpaces.dev)
+  return (
+    <GridRow
+      allStyle={[standardStyles.elementalMarginBottom, styles.alignOut]}
```

Comment:
It's probably a negligible improvement but in the app we avoid style arrays where we can b.c. they create new instances each render, breaking react's diff checking and forcing a rerender.
Instead we would destructure `elementalMarginBottom` in `styles.alignOut` like
```
...standardStyles.elementalMarginBottom,
other styles
```

Reply:
oh interesting that makes sense

Reply:
when i did this 
```
const styles = Stylesheet.create({
nameOfStyle: {
...standardStyles.centered,
width: 100
}}
```
`style={styles.nameOfStyle}`

the centered styles were not applied. this happened every time i spread a style